doc/classes/InputEvent.xml

```diff
@@ -4,6 +4,7 @@
 		Generic input event
 	</brief_description>
 	<description>
+		Base class of all sort of input event.
```

Comment:
Not a good sentence.  Also, it's not entirely accurate to call it a "base class" in that the built-in nature of the class hides that from the end user. The original `brief_description` was closer, it is a "generic input event".